### PR TITLE
R package arm

### PR DESCRIPTION
**Pull request by ThimotheeV**
New conda package for Data Analysis Using Regression and Multilevel/Hierarchical.

### CODE REVIEW

**Comment by bgruening** [comment]
The OSX error is actually because the `r-nloptr` is coming from the `defaults/r` channel. 
https://travis-ci.org/conda-forge/staged-recipes/builds/260939249#L259

If we can fix this, OSX should work as well.
ping @jakirkham

File: recipes/r-arm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+source:
+  fn: arm_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/arm_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/arm/arm_{{ version }}.tar.gz
+  sha256: 1cefcc1d82431f1b3ed014ff04dd087fd0da3396df5a25144ffa80a8d01fe700
+
+build:
+  number: 0
+  skip: true  # [win,osx]
```
Comment (ocefpaf) [inline recipes/r-arm/meta.yaml]: For readability's sake I am changing this to `[win or osx]`

**Comment by ocefpaf** [comment]
@ThimotheeV and @bgruening do you want to try to fix `OS X` now or should I merge this and you guys can sort it out in the feedstock?

**Comment by bgruening** [comment]
@ocefpaf merging now is ok. We can try to build OSX on the feedstock. But do you have an idea why this, only only this one package, is coming from defaults?

**Comment by ThimotheeV** [comment]
@ocefpaf  Same as @bgruening.

**Comment by ocefpaf** [comment]
> But do you have an idea why this, only only this one package, is coming from defaults?

Nope. But you are not alone :smile: 

https://github.com/conda-forge/r-rgdal-feedstock/issues/2

**Pull request merged**